sphinx/ext/todo.py

```diff
@@ -212,8 +218,14 @@ def depart_todo_node(self, node):
 
 def latex_visit_todo_node(self, node):
     # type: (nodes.NodeVisitor, todo_node) -> None
-    title = node.pop(0).astext().translate(tex_escape_map)
-    self.body.append(u'\n\\begin{sphinxadmonition}{note}{%s:}' % title)
+    target = node.get('targetref')
+    title = node[0].astext().translate(tex_escape_map)
```

Comment:
So far, the first child node is popped out. I don't know why that is needed. But this PR removes it. Please let me know the reason.

Reply:
Good catch. This was unintentional. Curiously it seems to have no ill effects. I confess, I don't understand the pop logic either. `Todo.run` returns two similar nodes, which then appear to be depended on in `process_todo`. Without any further information, I've just reverted to popping the node as it was done before.